lib/template/templates.c

```diff
@@ -319,6 +319,24 @@ log_template_options_init(LogTemplateOptions *options, GlobalConfig *cfg)
   options->initialized = TRUE;
 }
 
+void
+log_template_options_clone(LogTemplateOptions *source, LogTemplateOptions *dest)
+{
+  dest->ts_format = source->ts_format;
+  for (gint i = 0; i < LTZ_MAX; i++)
+    {
+      if (source->time_zone[i])
+        dest->time_zone[i] = g_strdup(source->time_zone[i]);
+    }
```

Comment:
Shouldn't we clone `time_zone_info` too?

Reply:
No, this has been a while ago, but I remember that this clone function clones the options structure in an "uninitialized state", so these timezone settings would be used to instantiate the TimeZoneInfo class, meaning that those would be unshared, but I don't feel that's a problem in this case.